### PR TITLE
Add `modality` to docstring, update `Parameters`

### PR DESCRIPTION
**Pull request by hoechenberger**
PR Description
--------------

Add `modality` to docstring, update `Parameters` section

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/mne-tools/mne-bids/blob/master/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"
- [ ] Commit history does not contain any merge commits

### CODE REVIEW

File: mne_bids/path.py [hunk 1]
```diff
@@ -67,17 +67,17 @@ class BIDSPath(object):
...
 
     Attributes
     ----------
-    check : bool
-        If ``True``, enforces the entities to be valid according to the
-        BIDS specification. The check is performed on instantiation
-        and any ``update`` function calls (and may be overridden in the
-        latter).
+    All parameters supplied to ``BIDSPath`` upon creation are exposed as
+    attributes with the same name, i.e., ``.subject``, ``.session``, etc.
```
Comment (agramfort) [inline mne_bids/path.py:80]: I prefer to reconsider this PR after the one from @adam2392 that touches the same lines.

Comment (hoechenberger) [reply mne_bids/path.py:80]: Right, let's just leave this one open

Comment (adam2392) [reply mne_bids/path.py:80]: I added this in, so I think this could be closed now.

**Comment by hoechenberger** [comment]
Closing as @adam2392 is taking over in another PR!

**Pull request closed without merging**